server/models/User.js

```diff
@@ -0,0 +1,40 @@
+const { Schema, model } = require('mongoose');
+const { ENUM } = require('sequelize/types');
```

Comment:
I think this should be lowercase, `enum`, to match how it appears on line 24 below